jetty-server/src/main/java/org/eclipse/jetty/server/Response.java

```diff
@@ -722,6 +722,11 @@ public boolean isWriting()
         return _outputType == OutputType.WRITER;
     }
 
+    public boolean isStreaming()
```

Comment:
What is the purpose of this?
This is named strangely to me.
Isn't OutputType.STREAM the one you should be checking for? (not WRITER)

Reply:
Oops yes - it should be checking from STREAM!